server/middleware/respond.js

```diff
@@ -7,23 +7,28 @@ const finishModel = (model, defaultTitleIntro) => {
 	if (!model.concept) {
 		return Object.assign({}, {
 			title: model.title,
-			titleHref: model.titleHref
+			titleHref: model.titleHref,
+			contentSelection: model.contentSelection,
 		}, listObj);
 	}
 
 	return Object.assign({}, {
 		title:  model.title || `${defaultTitleIntro} ${model.concept.preposition} ${model.concept.prefLabel}`,
 		titleHref:  model.titleHref || model.concept.relativeUrl,
-		concept: model.concept
+		concept: model.concept,
+		contentSelection: model.contentSelection,
 	}, listObj);
 };
 
+const sum = (total, value) => total + value;
+const numItems = (slot) => !slot ? 0 : !slot.items ? 0 : slot.items.length;
```

Comment:
Alternative to avoid double ternary operators, clarly a preference
```suggestion
const numItems = (slot) => slot && slot.items && slot.items.length || 0
```